package.json

```diff
@@ -9,17 +9,24 @@
   "author": "Fabian 'fabiantheblind' Morón Zirfas <fabian.moron.zirfas@gmail.com> (http://fabianmoronzirfas.me)",
   "license": "MIT",
   "dependencies": {
-    "clipboardy": "^1.1.0",
-    "commander": "^2.9.0",
-    "to-markdown": "^3.0.4"
+    "clipboardy": "^1.2.3",
+    "commander": "^2.18.0",
+    "turndown": "^5.0.1",
+    "turndown-plugin-gfm": "^1.0.2"
   },
   "preferGlobal": true,
   "bin": {
     "html2md": "./index.js"
   },
   "repository": "fabianmoronzirfas/to-markdown-cli",
   "devDependencies": {
-    "cz-conventional-changelog": "^2.0.0"
+    "cz-conventional-changelog": "^2.1.0",
+    "eslint": "^5.5.0",
+    "eslint-config-standard": "^12.0.0",
```

Comment:
I'm actually not happy with these plugins. Are they needed? 
I can give a example of my usual eslint config if you like.